pages/unreal-sdk/getting-started/before-you-start.md

```diff
@@ -29,37 +29,37 @@ Currently, the SDK supports the following functionality for Unreal Engine:
 
 ## SDK Installation
 
-1. Download the `AnkrSDK.zip` package from the latest [release](https://github.com/Ankr-network/game-unreal-sdk/releases).
-2. Unzip the `AnkrSDK.zip` package to your Unreal Project's **Plugins** folder.
+1. Download the `MirageSDK.zip` package from the latest [release](https://github.com/Ankr-network/game-unreal-sdk/releases).
```

Comment:
before we can update that we should change the unreal SDK first.

Reply:
It is already renamed to MirageSDK in the Unreal SDK repo latest releases, so this change makes it up to date